doc/index.rst

```diff
@@ -622,6 +622,23 @@ Functions
 
    .. versionadded:: 4.2.0
 
+.. function:: get_protocol_members(tp)
+
+   Return the set of members defined in a :class:`Protocol`.
```

Comment:
This links to our version of `Protocol`; it might be worth explicitly mentioning that the function works with `typing.Protocol` as well (same with `is_protocol` below)